projects/plugins/jetpack/extensions/blocks/donations/view.js

```diff
@@ -1,12 +1,14 @@
 import formatCurrency from '@automattic/format-currency';
 import domReady from '@wordpress/dom-ready';
-import { ENTER } from '@wordpress/keycodes';
 import { addQueryArgs, removeQueryArgs } from '@wordpress/url';
 import { minimumTransactionAmountForCurrency, parseAmount } from '../../shared/currencies';
 import { initializeMembershipButtons } from '../../shared/memberships';
 
 import './view.scss';
 
+// Enter keycode.
+const ENTER = 13;
```

Comment:
Even better, if you feel up to it, might be to switch from using [the deprecated event.keyCode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) to [event.code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) (or [event.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key)) that uses the string 'Enter' instead of a cryptic number.